repo/packages/G/gitlab/3/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "packagingVersion": "3.0",
+  "name": "gitlab",
+  "version": "1.0-8.14.1",
+  "minDcosReleaseVersion" : "1.8",
+  "scm": "https://gitlab.com/gitlab-org/gitlab-ce",
+  "maintainer": "support@gitlab.com",
+  "website": "https://about.gitlab.com/",
+  "framework": false,
+  "description": "Collaboration and source control management: code, test, and deploy together!",
```

Comment:
Add a link to dcos/example at the end of `description`. 
Something like: `Installation Guide: https://github.com/dcos/examples/tree/master/1.8/gitlab`